includes/plugins/woocommerce-subscriptions/class-subscriptions-confirmation.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * WooCommerce Subscriptions Confirmation class for FTC compliance.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Main class.
+ */
+class Subscriptions_Confirmation {
+
+	/**
+	 * Initialize hooks and filters.
+	 */
+	public static function init() {
+		if ( ! WooCommerce_Subscriptions::is_active() ) {
```

Comment:
```suggestion
		if ( ! class_exists( 'WooCommerce_Subscriptions' ) || ! WooCommerce_Subscriptions::is_active() ) {
```

Reply:
Yes! Fixed in [d009e0f](https://github.com/Automattic/newspack-plugin/pull/3961/commits/d009e0fcf2de1cbff0d0f70d2c606c8cd1aec853).